object/service.proto

```diff
@@ -10,14 +10,14 @@ import "refs/types.proto";
 import "session/types.proto";
 
 // `ObjectService` provides API for manipulating objects. Object operations do
-// not interact with sidechain and are only served by nodes in p2p style.
+// not address sidechain and are only served by nodes in p2p style.
```

Comment:
Not sure if it's the right verb here.

Reply:
I just dislike `operations interact with`, I would change the verb after `operations`